entrypoint.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+cp -n /app/default-configs/* /app/configs/
```

Comment:
Why copy in the entrypoint? Can you remove this and put master at the ENTRYPOINT directly?

Reply:
this puts the files on disk. Which will be mounted, and make these files persistent.
This gives users the option to edit the configs on disk instead of using env vars if they so wish.

Same with vanity, allows users to edit them on the mounted disk.

I'm not sure if there's a better way to do this.
But I always like when images do this and I don't need to go manually grab default configs from whereever to put on my host disk

Reply:
Some thoughts:
- I would argue this cp -n is worse than none by default because it doesn't diff, so once you upte you have stale config sitting there preventing a copy and you need to merge by hand again. I do see the appeal of default config files showing up on their own though.
- I'm worried that this is another place added to the already complex master startup that does magic. For k8s for example I need to rely on being able to replace entrypoint with WorldServers so this script must not have load bearing (fatal when skipped) side effects

Reply:
Stale configs is just part of it unfortunately. There is no remedy other than making the server possibly update the configs with new setting during startup and providing a sane default (which we could do that since we'll have the latest default configs in the container always)

All the necessary configs already exist and any new configs should only be superfluous with the default behavior that the setting would change being what happened before it was added.

This is 100% just a way to get a bootstrap on a fresh container.
Users of docker know they can modify the entrypoint in their run command if they need to do so, which I've personally rarely had to do. I'm assuming you're already modifying the entrypoint in k8's since you're deploying individual servers to separate pods (I think that terminology is correct)

Reply:
> There is no remedy other than making the server possibly update the configs with new setting during startup and providing a sane default

Having the configs be writable may be a concern as well, but that's a different discussion imo

Also, could just forsake this all and force the user to either do everything via env vars or know they need to bind mount the configs and the go grab them from the repo

And we should probably remove the dependency for the configs files needing to exist at all as well, but that can be a separate PR.

This is all just to get a MVP for docker without any server changes